backend/package.json

```diff
@@ -3,6 +3,11 @@
   "version": "1.0.0",
   "description": "",
   "main": "src/index.ts",
+  "nodemonConfig": {
+    "ignore": [
+      "src/utils/swagger-output.json"
+    ]
+  },
```

Comment:
Why this is necessary

Reply:
During dev we use nodemon to watch for changes right? 
It creates an infinite loop as every time the swagger-output is generated, it assumes it to be a change and restarts the server over and over again. 
Thats why this has to be present

Reply:
OK